README.md

```diff
@@ -557,9 +557,11 @@ If you'd like `just` to default to listing the recipes in the `justfile`, you ca
 
 ```make
 default:
-  @just --list
+  @just -f {{justfile()}} --list
```

Comment:
I like using use long options in documentation and examples, since it's clearer than short:

```suggestion
  @just --justfile {{justfile()}} --list
```

Reply:
I prefer long options, too, but made the wrong assumptions about your preference. :)